app/src/main/java/com/douglasbruce/tasky/features/event/EventScreen.kt

```diff
@@ -103,6 +106,17 @@ internal fun EventRoute(
         }
     }
 
+    val snackbarHostState = remember { SnackbarHostState() }
+    val context = LocalContext.current
+    LaunchedEffect(key1 = snackbarHostState) {
+        viewModel.infoMessages.collect { message ->
+            snackbarHostState.showSnackbar(
+                message = message.asString(context),
+                withDismissAction = true,
+            )
+        }
+    }
```

Comment:
Strictly speaking, you need something like this to collect one-time events 100% safely without risking to lose some:
```
@Composable
private fun <T> ObserveAsEvents(flow: Flow<T>, onEvent: (T) -> Unit) {
    val lifecycleOwner = LocalLifecycleOwner.current
    LaunchedEffect(flow, lifecycleOwner.lifecycle) {
        lifecycleOwner.repeatOnLifecycle(Lifecycle.State.STARTED) {
            withContext(Dispatchers.Main.immediate) {
                flow.collect(onEvent)
            }
        }
    }
}
```
For more info, I talk about the problem in this video: https://youtu.be/njchj9d_Lf8?si=0to4KwRHzEIECID8